manifests/analytics/install.pp

```diff
@@ -4,23 +4,34 @@
 #
 # === Parameters:
 #
-# [*package_name*]
-#   (optional) Package name for analytics
+# [*container_name*]
+#   (optional) Container name to load,
 #
+# [*container_url*]
+#   Mandatory for container based deployment
+#   URL for downloading container
+#
+
 class contrail::analytics::install (
-) {
+  $container_image          = undef,
+  $container_name           = undef,
+  $container_url            = undef,
+) inherits contrail::params {
 
-  package { 'wget' :
-    ensure => latest,
+  if $version < 4 {
+    package { 'python-redis' :
+      ensure => absent,
+    } ->
+    package { 'python-gevent' :
```

Comment:
I remember there was a version dependency with python-gevent. Nagendra can probably comment on this one.